tests/integration/api_v3/http/transactionsDryRun.test.js

```diff
@@ -47,35 +48,56 @@ const endpoint = `${baseUrlV3}/transactions/dryrun`;
 const postTransactionEndpoint = `${baseUrlV3}/transactions`;
 
 describe('Post dryrun transactions API', () => {
-	it('Returns proper response (Fail) when transaction object has less than required fee', async () => {
+	it('should return proper response (Fail) when transaction object has less than required fee', async () => {
 		const response = await api.post(endpoint, { transaction: TRANSACTION_OBJECT_PENDING });
 		expect(response).toMap(goodRequestSchemaFortransactionsDryRun);
 		expect(response.data).toMap(dryrunTransactionPendingResponseSchema);
 		expect(response.meta).toMap(metaSchema);
 	});
 
-	it('Returns proper response (Fail) when transaction string has less than required fee', async () => {
+	it('should return proper response (Fail) when transaction string has less than required fee', async () => {
 		const response = await api.post(endpoint, { transaction: TRANSACTION_ENCODED_PENDING });
 		expect(response).toMap(goodRequestSchemaFortransactionsDryRun);
 		expect(response.data).toMap(dryrunTransactionPendingResponseSchema);
 		expect(response.meta).toMap(metaSchema);
 	});
 
-	it('Post dryrun transaction succesfully with only transaction object', async () => {
+	it('should post dryrun transaction succesfully with only transaction object', async () => {
 		const response = await api.post(endpoint, { transaction: TRANSACTION_OBJECT_VALID });
 		expect(response).toMap(goodRequestSchemaFortransactionsDryRun);
 		expect(response.data).toMap(dryrunTransactionSuccessResponseSchema);
 		expect(response.meta).toMap(metaSchema);
 	});
 
-	it('Post dryrun transaction succesfully with only transaction string', async () => {
+	it('should post dryrun transaction succesfully with only transaction string', async () => {
 		const response = await api.post(endpoint, { transaction: TRANSACTION_ENCODED_VALID });
 		expect(response).toMap(goodRequestSchemaFortransactionsDryRun);
 		expect(response.data).toMap(dryrunTransactionSuccessResponseSchema);
 		expect(response.meta).toMap(metaSchema);
 	});
 
-	it('Post dryrun transaction succesfully with only transaction skipping verification', async () => {
+	it('should return proper response (Success) when posting unsigned transaction with strict: false', async () => {
+		const response = await api.post(
+			endpoint,
+			{ transaction: UNSIGNED_TRANSACTION_OBJECT, strict: false },
+		);
+		expect(response).toMap(goodRequestSchemaFortransactionsDryRun);
```

Comment:
would it be better to refactor the variable name to use proper camel casing?

```suggestion
		expect(response).toMap(goodRequestSchemaForTransactionsDryRun);
```